pkg/leclient/lets_encrypt.go

```diff
@@ -233,18 +232,6 @@ func getLetsEncryptAccountSecret(kubeClient client.Client) (secret *v1.Secret, e
 	secretName := letsEncryptAccountSecretName
 
 	secret, err = GetSecret(kubeClient, secretName, config.OperatorNamespace)
-	if err != nil {
```

Comment:
so we're not checking if this errors out at all? is this intentional?

Reply:
correct - i don't think there's any error handling to be done here, so any error will be returned to where `getLetsEncryptAccountSecret()` is called for handling at that point

if there's some handling we can do at this point, i'm happy to add it though